src/main/webapp/js/omniComment.js

```diff
@@ -402,6 +410,16 @@ function enableComment(commentIdx) {
     $('#plainCommentText' + commentIdx).hide();
     $("div[id='commentTextEdit" + commentIdx + "']").show();
     $("textarea[id='commentText" + commentIdx + "']").val($('#plainCommentText' + commentIdx).text());
+
+    if (typeof richTextEditorBuilder !== 'undefined') {
+        /* eslint-disable camelcase */ // The property names are determined by external library (tinymce)
+        richTextEditorBuilder.initEditor('#commentText' + commentIdx, {
+            inline: true,
+            fixed_toolbar_container: '#rich-text-toolbar-comment-container-' + commentIdx
+        });
+        /* eslint-enable camelcase */
+    }
```

Comment:
I think it can't. It's already a routine in richTextEditor.js and it has variable params like 'fixed_toolbar_container' and selector